src/models/rfcProps.js

```diff
@@ -220,35 +219,27 @@ const properties = {
 			value: '',
 			type: ['facebook'],
 		},
-		info: t(
-			'contacts',
-			'The link of the profile. e.g. https://www.facebook.com/Nextclouders/ '
-		),
 		options: [
-			{ id: 'FACEBOOK', name: 'Facebook' },
-			{ id: 'GITHUB', name: 'GitHub' },
-			{ id: 'GOOGLEPLUS', name: 'Google+' },
-			{ id: 'INSTAGRAM', name: 'Instagram' },
-			{ id: 'LINKEDIN', name: 'LinkedIn' },
-			{ id: 'PINTEREST', name: 'Pinterest' },
-			{ id: 'QZONE', name: 'QZone' },
-			{ id: 'TUMBLR', name: 'Tumblr' },
-			{ id: 'TWITTER', name: 'Twitter' },
-			{ id: 'WECHAT', name: 'WeChat' },
-			{ id: 'YOUTUBE', name: 'YouTube' },
-			{ id: 'MASTODON', name: 'Mastodon' },
-			{ id: 'DIASPORA', name: 'Diaspora' },
-			{ id: 'OTHER', name: 'other Social Media' },
+			{ id: 'FACEBOOK', name: 'Facebook', placeholder: 'https://facebook.com/…' },
+			{ id: 'GITHUB', name: 'GitHub', placeholder: 'https://github.com/…' },
+			{ id: 'GOOGLEPLUS', name: 'Google+', placeholder: 'https://plus.google.com/…' },
+			{ id: 'INSTAGRAM', name: 'Instagram', placeholder: 'https://instagram.com/…' },
+			{ id: 'LINKEDIN', name: 'LinkedIn', placeholder: 'https://linkedin.com/…' },
+			{ id: 'PINTEREST', name: 'Pinterest', placeholder: 'https://pinterest.com/…' },
+			{ id: 'QZONE', name: 'QZone', placeholder: 'https://qzone.com/…' },
+			{ id: 'TUMBLR', name: 'Tumblr', placeholder: 'https://tumblr.com/…' },
+			{ id: 'TWITTER', name: 'Twitter', placeholder: 'https://twitter.com/…' },
+			{ id: 'WECHAT', name: 'WeChat', placeholder: 'https://wechat.com/…' },
+			{ id: 'YOUTUBE', name: 'YouTube', placeholder: 'https://youtube.com/…' },
+			{ id: 'MASTODON', name: 'Mastodon', placeholder: 'https://mastodon.social/…' },
+			{ id: 'DIASPORA', name: 'Diaspora', placeholder: 'https://joindiaspora.com/…' },
+			{ id: 'OTHER', name: 'Other social media', placeholder: 'https://example.com/…' },
```

Comment:
The placeholders did not show up when I tried it out?
![placeholder](https://user-images.githubusercontent.com/34400929/88977391-1b307080-d2be-11ea-9118-0b72685b078b.gif)

And because you asked: I find it easier to read when only the username is stored, not the URL. In my point of view that is redundant information to the type and just fills up the complete textfield.
![image](https://user-images.githubusercontent.com/34400929/88978033-394aa080-d2bf-11ea-9f7e-7b393dd17138.png)